tingbot/platform_specific/sdl_wrapper.py

```diff
@@ -49,6 +49,7 @@ def click(self, xy, action):
             self.surface.fill(self.color)
             if button_callback:
                 button_callback(self.number, 'up')
+        screen.needs_update = True
 
```

Comment:
Oh, I get it. There's something a bit strange about talking to the little screen to update something outside of it - could this be changed to `self.surface.update()`?